mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelFactory.kt

```diff
@@ -101,11 +101,13 @@ data class FragmentViewModelContext(
     /**
      * The fragment owner of the ViewModel.
      */
-    val fragment: Fragment
+    val fragment: Fragment,
+    private val ownerProvider: ViewModelStoreOwner = fragment,
+    private val savedStateRegistryProvider: SavedStateRegistry = fragment.savedStateRegistry
```

Comment:
Shouldn't this be `private val savedStateRegistryProvider: SavedStateRegistry = ownerProvider.savedStateRegistry`? Just wondering what would happen If you assign an `ownerProvider` but not a `savedStateRegistryProvider` .